appendix-basic65.tex

```diff
@@ -187,30 +192,57 @@ \subsection{BASIC 65 Constants}
 \subsection{BASIC 65 Variables}
 \index{BASIC 65 Variables}
 
-Each scalar variable consumes 8 bytes of storage in memory.
-The reserved area in bank 0 from \$F700-\$FEFF can store 256 variables.
```

Comment:
Yes, this will go in a chapter on BASIC memory use.

There is an argument that this belongs in a description of the BASIC API because there is a (new) BASIC keyword that returns the memory location of a variable, so describing the data structure that it points to is necessary to make that keyword useful. But this is an introductory (and cursory) tour of BASIC features, so it doesn't belong here.